spec/order_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'order'
+
+describe Order do
+  subject(:order) {  described_class.new '2 Spring Rolls, 1 Crispy Duck', 12.97 }
```

Comment:
Metrics/LineLength: Line is too long. [81/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)